packages/sns/src/governance.canister.spec.ts

```diff
@@ -967,7 +967,7 @@ describe("Governance canister", () => {
     };
     const topic1: Topic = { DappCanisterManagement: null };
     const topic2: Topic = { Governance: null };
-    const alias: string = "alias";
```

Comment:
in theory this is less restrictive, but I find it better :)

Reply:
Agree 👍